assignment2.txt

```diff
@@ -0,0 +1,9 @@
+Lloyd Page
+1. Two or more operators can be combined into a single line of code
+2. logical errors are valid programs with unexpected outcomes, syntax errors break the program, causing it to crash.
+3. True
```

Comment:
Remember that there are no boolean values in C, so the answer is technically true!